Client/Frontend/Home/TopSites/HomeTopSite.swift

```diff
@@ -38,9 +47,9 @@ final class HomeTopSite {
     init(site: Site, profile: Profile) {
         self.site = site
         if let provider = site.metadata?.providerName {
-            title = provider.lowercased()
+            title = provider.lowercased().capitalized
         } else {
-            title = site.tileURL.shortDisplayString
+            title = site.title
```

Comment:
this case doesn't need to be capitalized?

Reply:
It's already capitalized properly. Like the jump back in cell has